.buildkite/shared-pipeline-vars

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
@mokagio it seems we don't use it. Should we remove it? Or did you intend to refactor pipeline to use this file?

Reply:
@wojtekn 

This file is `source`'d before calling `buildkite-agent pipeline upload`, and can be used to set up some variables that will be interpolated in the `.yml` pipeline before uploading it.

See https://github.com/Automattic/buildkite-ci/blob/c0d6ade0cc1b7dd5e8cb69beaf1161a28a679156/src/buildkite/pipelines/studio.tf#L22

Maybe I could updated the message below with something like "in CI" or "by CI".